docusaurus/docs/learn/quickstarts/quickstarts.md

```diff
@@ -45,3 +45,7 @@ access to your [network](/learn/introduction/index.mdx) and aren't on the local
 * [Run Everything Locally - using Docker](./network/local-with-docker.md)
 * [Run Everything Locally - Docker Compose](./network/local-docker-compose.md)
 * [Run Everything Hosted](./network/hosted.md)
+
+## Production Deployments
+
+For production deployments, see the [deployment guides](/docs/category/deployments).
```

Comment:
I'd really like to see this comment made higher in the page. Like right at the top. Something that clearly says quickstarts are intended for quick learning / dev mode and not particularly intended for long-term support. See "production deployments" for deploying a long-running overlay (something like that)